packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -2742,4 +2742,34 @@ export default class Meeting extends StatelessWebexPlugin {
   unlockMeeting() {
     return MeetingUtil.unlockMeeting(this.inMeetingActions, this.meetingRequest, this.locusUrl);
   }
+
+  /**
+   * Sends DTMF tones to the current meeting
+   * @param {String} tones a string of one or more DTMF tones to send
+   * @returns {Promise}
+   * @public
+   * @memberof Meeting
+   */
+  sendDtmf(tones) {
+    const reject = (message) => {
+      LoggerProxy.logger.error(message);
+
+      return Promise.reject(new Error(message));
+    };
+
+    if (this.locusInfo && this.locusInfo.self) {
+      if (this.locusInfo.self.enableDTMF) {
+        return this.meetingRequest
+          .sendDtmf({
+            locusUrl: this.locusInfo.self.url,
+            deviceUrl: this.deviceUrl,
+            tones
+          });
+      }
+
+      return reject('Meeting:index#sendDtmf --> cannot send dtmf, meeting does not have dtmf enabled');
+    }
+
+    return reject('Meeting:index#sendDtmf --> cannot send dtmf, meeting does not have a locus. Have you joined?');
```

Comment:
Not sure if third-parties would be using this, but saying "doesn't locus" would make sense for the average Joe?
Also, do you mind capitalizing DTMF as it's an acronym? thanks